src/ts/components/seekbar.ts

```diff
@@ -207,46 +207,62 @@ export class SeekBar extends Component<SeekBarConfig> {
     let isUserSeeking = false;
     let isPlayerSeeking = false;
 
+    const getPlaybackPositionPercentage = (): number => {
+      if (player.isLive()) {
```

Comment:
I guess we could pass this in as a parameter. 

Or split it up even more and have 3 functions: 

```
getLivePlaybackPositionPercentage() {return getPlaybackPositionPercentage(true)}
getVodPlaybackPositionPercentage() {return getPlaybackPositionPercentage(false)}
getPlaybackPositionPercentage(isLive: boolean) {
... the stuff
}
```

Reply:
For this case, I do not think this is necessary.

A bigger refactoring needs to be done here and move more of the `playbackPositionHandling` into a separate file, but currently, that is out of scope for this PR.

Reply:
👍